.github/workflows/pr.yml

```diff
@@ -0,0 +1,101 @@
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Cache Operator SDK ${{ env.OPERATOR_SDK_VERSION }}
+        uses: actions/cache@v2
+        id: cache-operator-sdk
+        with:
+          path: ~/cache
+          key: operator-sdk-${{ env.OPERATOR_SDK_VERSION }}
+      - name: Download fedora environment dependencies # Would be nice to cache this somehow
+        run: sudo dnf -y install diffutils findutils gcc wget
+      - name: Download Operator SDK ${{ env.OPERATOR_SDK_VERSION }}
+        if: steps.cache-operator-sdk.outputs.cache-hit != 'true'
+        run: |
+          mkdir -p ~/cache
+          wget https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk_linux_amd64 -O ~/cache/operator-sdk-${OPERATOR_SDK_VERSION} > /dev/null -O ~/cache/operator-sdk-${OPERATOR_SDK_VERSION} > /dev/null
+          chmod +x ~/cache/operator-sdk-${OPERATOR_SDK_VERSION}
+      - name: Install Operator SDK ${{ env.OPERATOR_SDK_VERSION }}
+        run: |
+          mkdir -p ~/bin
+          cp ~/cache/operator-sdk-${OPERATOR_SDK_VERSION} ~/bin/operator-sdk
+          echo "$HOME/bin" >> $GITHUB_PATH
+      - name: Cache go modules
+        id: cache-mod
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Download dependencies
+        run: go mod download
+        if: steps.cache-mod.outputs.cache-hit != 'true'
+      - name: Check go mod status
+        run: |
+          time_before=$(date "+%F %T")
+          go mod tidy
+          if [[ $(find . -type f -newermt "$time_before") ]]
+          then
+            echo "Go mod state is not clean:"
+            find . -type f -newermt "$time_before"
+            exit 1
+          fi
+      - name: Check format
+        run: |
+          time_before=$(date "+%F %T")
+          make fmt
+          if [[ $(find . -type f -newermt "$time_before") ]]
+          then
+            echo "Some files are not properly formatted."
+            echo "Please run `go fmt` and amend your commit."
+            find . -type f -newermt "$time_before"
+            exit 1
+          fi
+      - uses: dominikh/staticcheck-action@v1.1.0
+        with:
+          version: "2021.1.2"
+          install-go: false
+      - name: Check manifests # make generate manifests touches files even for no changes
+        run: |
+          cp -rp ../release-service/ /tmp/release-service/
+          make generate manifests
+          if [[ $(diff -r ../release-service /tmp/release-service) ]]
+          then
+            echo "generated sources are not up to date:"
+            diff -r ../release-service /tmp/release-service
+            exit 1
+          fi
+      - name: Run Go Tests
+        run: |
+          make test
+  docker:
+    name: Check docker build
+    runs-on: ubuntu-latest
```

Comment:
Use ubuntu-20.04 or change the previous one (line 11) to use ubuntu-latest.

Reply:
Good catch